### ISSUE TITLE
Sprint 27 a11y support

### ISSUE THREAD

**Issue by LynneJohnson224** [department-of-veterans-affairs/va.gov-team#78288]
## User Story
As a shared support team member, I need to understand a11y needs each MHV product team has so I can ensure we align with design system requirements and provide the best user experience possible for Veterans and other users of VA.gov. 

## Description
Any MHV teams including: Secure Messaging, Medical Records, Medications, and also related Appointments & Check-In teams may need to request shared support services for accessibility. Within a given sprint, that work should be articulated through tasks (below) and updates (comments on the ticket) as needed. 

### Ongoing tasks: 
- [ ] 

### New tasks this sprint
- [ ] Help teams and Bobby with design system V3 components
- [ ] Prep for upcoming research sessions and assisting with AT (assistive tech)

**Comment by danbivins** [comment]
Generally the two tasks above will be continuing to next sprint.
